articles/TOC.yml

```diff
@@ -1,44 +1,52 @@
 - name: Microsoft Quantum Documentation
   href: index.yml
-- name: Overview
+- name: Getting started
   items:
-  - name: Get started
+  - name: What is quantum computing?
+    href: overview/what-is-quantum-computing.md
+  - name: What can a quantum computer do?
+    href: overview/quantum-computers.md
+  - name: Why learn quantum computing?
+    href: overview/why-learn-quantum-computing.md
+  - name: What are Q# and QDK?
+    href: overview/what-is-qsharp.md
+  - name: How can I learn more?
+    href: overview/how-to-learn-quantum-computing.md
+- name: The QDK introduction
+  items:
+  - name: Get started with the Quantum Development Kit
     href: welcome.md
-  - name: Five questions about quantum computing
+  - name: Install and update
     items:
-    - name: What is quantum computing?
-      href: overview/what-is-quantum-computing.md
-    - name: What can a quantum computer do?
-      href: overview/quantum-computers.md
-    - name: Why learn quantum computing?
-      href: overview/why-learn-quantum-computing.md
-    - name: What is Q#?
-      href: overview/what-is-qsharp.md
-    - name: How to learn quantum computing?
-      href: overview/how-to-learn-quantum-computing.md
-- name: Install and update
+    - name: Install the QDK
+      items:
+      - name: Installation overview
+        href: install-guide/index.md
+      - name: Q# + C#
+        href: install-guide/csinstall.md
+      - name: Q# + Python
+        href: install-guide/pyinstall.md
+      - name: Q# Jupyter notebooks
+        href: install-guide/qjupyter.md
+    - name: Update the QDK
```

Comment:
I think we should change the "The QDK introduction" section to be more self-explanatory in its content. Negative feedback we received last semester was about people not finding easily the installation guide. 
I think "Install the QDK" or "Get the QDK" should appear as a main section on the TOC, and not behind a generic click-wall name. For example, if I'm a reader looking for the installation guide, why should I look into "The QDK introduction" and not in "Getting started"?

Reply:
Totally agree, and was going to say the same. I think "Get the QDK" would be much better, because the other page in there fits that